src/software/ai/navigator/path_planner/theta_star_path_planner_test.cpp

```diff
@@ -248,22 +217,15 @@ TEST(TestThetaStarPathPlanner, test_theta_star_path_planner_same_cell_dest)
     std::vector<Obstacle> obstacles = std::vector<Obstacle>();
 
     std::unique_ptr<PathPlanner> planner = std::make_unique<ThetaStarPathPlanner>();
-    std::vector<Point> path_points;
 
-    try
-    {
-        path_points = std::get<std::vector<Point>>(
-            planner->findPath(start, dest, field, obstacles));
-    }
-    catch (const std::bad_variant_access&)
-    {
-        // not a valid path
-        FAIL();
-    }
+    Rectangle navigableArea(Point(field.totalXLength(), field.totalYLength()),
+                            field.totalXLength(), field.totalYLength());
 
-    ASSERT_EQ(2, path_points.size());
-    ASSERT_EQ(start, path_points.front());
-    ASSERT_EQ(dest, path_points.back());
+    Path path = planner->findPath(start, dest, navigableArea, obstacles);
+
+    EXPECT_EQ(2, path.size() + 1);
```

Comment:
Nit: expect `path.size()` to be equal to `1` instead?

Reply:
done